frontend/src/scenes/insights/LineGraph/LineGraph.jsx

```diff
@@ -402,6 +402,16 @@ export function LineGraph({
             onClick: (_, [point]) => {
                 if (point && onClick) {
                     const dataset = datasets[point._datasetIndex]
+                    const crossDataset = datasets
+                        .filter((_dt) => !_dt.dotted)
+                        .map((_dt) => ({
+                            seriesId: _dt.id,
+                            action: _dt.action,
+                            personUrl:
+                                typeof point._index !== 'undefined' ? _dt.persons_urls?.[point._index].url : undefined,
+                            breakdown_value: _dt.breakdown_value,
+                            pointValue: typeof point._index !== 'undefined' ? _dt.data[point._index] : undefined,
+                        }))
```

Comment:
The new refactor of `LineGraph` has a more robust way of capturing which datasets are being clicked on. The `onClick` callback will now have `pointsIntersectingLine` and `pointsIntersectingClick` which are more robustly calculated by `chart.js's` `Chart.getElementsAtEventForMode()` function. I think you can leverage that to achieve the same effect here.